traits/trait_dict_object.py

```diff
@@ -53,8 +53,12 @@ def __init__(self, *, removed=None, added=None, changed=None):
         self.changed = changed
 
     def __repr__(self):
-        return "TraitDictEvent(removed={!r}, added={!r}, changed={!r})".format(
-            self.removed, self.added, self.changed
+        return (
+            "{event.__class__.__name__}("
+            "removed={event.removed!r}, "
+            "added={event.added!r}, "
+            "changed={event.changed!r}"
+            ")".format(event=self)
```

Comment:
nit : this might look cleaner if we use an f-string.

Reply:
Agreed; and the `event` -> `self` switch confused me the first time I looked at this code - just use `self` throughout.

Reply:
> Agreed; and the `event` -> `self` switch confused me the first time I looked at this code - just use `self` throughout.

I had originally been following what was done here:
https://github.com/enthought/traits/blob/7f8546d851fc476fe287f6f942afe13807557da3/traits/observation/_dict_change_event.py#L45-L52
and with the other `_something_change_event` reprs.  Shall I change these to just use self throughout and f strings as well?

Reply:
Ah, I see. Not sure it's worth changing that existing code, given that it's already working. Maybe do that in a separate PR?

Edit: Okay, fine!